app/src/main/res/layout/favorite_item.xml

```diff
@@ -18,11 +18,12 @@
 
     <ImageView
         android:id="@+id/item_notification_dot"
-        android:layout_width="14dp"
-        android:layout_height="14dp"
-        android:layout_gravity="end|top"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_gravity="center"
         android:contentDescription="@null"
         android:src="@drawable/notification_dot"
+        android:tint="#FF4caf50"
```

Comment:
Do you need the tint in the XML? It's applied dynamically at runtime, and I guess inflating it with a tint and changing it after is slightly more expensive?

It's not really a big deal performance wise, but if you want to keep it can we at least use a @resource rather than harcode the color?